src/config/transport/transport-strategy.ts

```diff
@@ -0,0 +1,85 @@
+import {IGrpcConfiguration} from './grpc-configuration';
+
+export interface ITransportStrategy {
+  getMaxConcurrentRequests(): number | null;
```

Comment:
Never used.

Reply:
good call, we don't actually have a way to limit these in JS for now, so I'll peel these out for the time being.